src/main/java/io/weaviate/client6/v1/api/collections/vectorizers/Text2MultiVecJinaAiVectorizer.java

```diff
@@ -15,6 +15,7 @@
 public record Text2MultiVecJinaAiVectorizer(
     @SerializedName("model") String model,
     @SerializedName("dimensions") Integer dimensions,
+    @SerializedName("baseURL") String baseURL,
```

Comment:
In Java abbreviations aren't spelled in all-caps (like they are in Go for example), so this should be `baseUrl`

(the variable name itself, ofc the SerializedName can be whichever name the API expects to get)